pkg/api/v1beta1/dynakube/convert_from.go

```diff
@@ -3,16 +3,16 @@ package dynakube
 import (
 	"strconv"
 
-	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
-	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube/oneagent"
+	dynakubev1beta4 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta4/dynakube"
+	oneagentv1beta4 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta4/dynakube/oneagent"
```

Comment:
Is this naming really necessary? 

You have for example: `HostInjectSpec{}` on the one hand and `oneagentv1beta4.HostInjectSpec{}`

Not that it matters much and would not mind if we keep it, just want to question it, as I am not used to reading it like that

Reply:
In `v1beta1` the `oneagent` sub package did not exist yet. In higher API versions you have conversions between e.g. `oneagent.HostInjectSpec` and `oneagentv1beta4.HostInjectSpec`. So there package aliases are needed anyway. And I want to see clearly to/from what the version the conversion is done.